### DISCUSSION TITLE
Potential in the future for upgrades

### DISCUSSION THREAD

**Discussion by DS1992ds** [CorsixTH/CorsixTH#2160] [Research Room]
hey all 

I am first going to say that I absolutely love the work that has gone into updating this classic game, I am constantly playing CorsixTH when I get time,  I don't think the coders get the appreciation they truly deserve 

I was thinking the other day how this now has so much potential, but not knowing coding I am unsure if the following is even possible, but can new diseases, rooms or equipment be created for this game? don't get me wrong I love the game with original content but wouldn't it be awesome if this was to be possible (like the original pregnancy patch, adding no graphics as such to the original game but was just a new disease) 

I know this will be a long way down the list of ongoing improvements or the "legality" side of things and again forgive my lack of knowledge in relation to coding/ possibilities :)

i look forward to the discussion which may follow, please don't shout at me for asking :P (and sorry if this has already been asked)

dan :)

**Comment by lewri** [comment]
Hello there. I wouldn't worry about coding ability or not, it's nice just to have people about willing to play and give feedback!

New diseases/rooms/equipment are definitely possible in the future. But, we would also need to make new graphics (and a system to support it).
However, at a stretch you could currently get away with making a new non-visual disease (since they use the standard patient sprites).

As for pregnancy, I do believe it is in the game but it gets represented as a standard female patient (though sometimes it gets disabled too).
